pkg/actuators/machine/instances.go

```diff
@@ -269,7 +269,7 @@ func launchInstance(machine *clusterv1.Machine, machineProviderConfig *providerc
 	}
 	tagVolume := &ec2.TagSpecification{
 		ResourceType: aws.String("volume"),
-		Tags:         []*ec2.Tag{{Key: aws.String("clusterid"), Value: aws.String(clusterID)}},
+		Tags:         rawTagList,
```

Comment:
Why `rawTagList` and not `tagList`? The latter appears to just be the raw list with duplicated removed.

Reply:
you're totally right. Updated, thanks!